portfolio/src/main/webapp/script.js

```diff
@@ -128,21 +128,28 @@ function showCommentInfo() {
   document.getElementById('comment-info').innerText= 'Log In to add and view comments';
 }
 
+/** Removes comments from Datastore. */
+function deleteComment(commentId) {
+  let deleteMessage;
+  if (commentId === 'all') {
+    deleteMessage = 'Do you really want to delete all the comments ?';
+  } else {
```

Comment:
Instead of checking for "all" here, just create a new method.

```javascript
deleteAllComments() {
  if (window.confirm("Are you sure that you want to delete all comments?")) {
    fetch('/comments', {method: 'DELETE'})
      ...
  }
}
```

Deleting one comment should be another end point.

`DELETE /comments/:id`

This may not be directly supported by the servlet framework but I think wildcards are supported

`DELETE /comments/*` then you can extract the path parameters.

Not really necessary.

Reply:
This seems like it needs me more time to research on this.
Should I try this approach in next PR for the time being ?